core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -105,6 +107,10 @@
     private int http1MaxChunkSize = Flags.defaultHttp1MaxChunkSize();
 
     // Armeria-related properties:
+    @Nullable
+    private Function<? super EventLoopGroup, ? extends EventLoopScheduler> eventLoopSchedulerFactory;
+    private int maxEventLoopPerEndpoint;
```

Comment:
`maxNumEventLoopsPerEndpoint` ? (`num`, like we have `maxNumConnections` / Use plural for `EventLoop`)

Reply:
Fixed. Thanks!